src/role-access/containers/specific-access/specific-access-duration/specific-access-duration.component.ts

```diff
@@ -0,0 +1,56 @@
+import { Component, Input, OnDestroy, OnInit } from '@angular/core';
+import { PersonRole } from '@hmcts/rpx-xui-common-lib/lib/models/person.model';
+import { Store } from '@ngrx/store';
+import { Subscription } from 'rxjs';
+import { ERROR_MESSAGE, PERSON_ROLE } from '../../../constants';
+import { SpecificAccessNavigationEvent, SpecificAccessState } from '../../../models';
+import { SpecificAccessNavigation } from '../../../models/specific-access-navigation.interface';
+import * as fromFeature from '../../../store';
+
+@Component({
+  selector: 'exui-specific-access-duration',
+  templateUrl: './specific-access-duration.component.html'
+})
+export class SpecificAccessDurationComponent implements OnInit, OnDestroy {
+  public ERROR_MESSAGE = ERROR_MESSAGE;
+  @Input() public navEvent: SpecificAccessNavigation;
+
```

Comment:
these don't appear to be needed?

Reply:
Removed. Was secretly hoping the person working on this would remove if they didn't require it to save adding it again.